src/uniprotkb/adapters/uniProtkbConverter.ts

```diff
@@ -72,6 +72,8 @@ export type UniProtkbAPIModel = {
   entryAudit?: EntryAudit;
   references?: Reference[];
   lineages?: LineageData[];
+  // How is that defined? What goes in this?
+  extraAttributes?: Record<string, unknown>;
```

Comment:
This is actually something we should be using for UniProtKB cards - we are currently getting pretty much all (or maybe all?) the domain to be able to count the number of items to display at the bottom of cards. This was added later by backend to prevent us from doing so and we should update it. I will create a Jira.